x/ibc/02-client/handler_test.go

```diff
@@ -13,12 +13,6 @@ func (suite *ClientTestSuite) TestHandleCreateClientLocalHost() {
 		msg      exported.MsgCreateClient
 		expPass  bool
 	}{
-		{
-			"localhost",
```

Comment:
localhost is already created in genesis. This test would fail, changing it to false would just make it a duplication of the later test case `"client already exists",`